src/Atmos/Model/turbulence.jl

```diff
@@ -63,7 +65,7 @@ end
 """
     SmagorinskyLilly <: TurbulenceClosure
 
-  § 1.3.2 in CliMA documentation 
+  § 1.3.2 in CliMA documentation
```

Comment:
Either update or remove the references to the design documents. These are no longer correct.

Reply:
I'll do this everywhere, we can't expect that they'll stay in sync.